kitchen.docker.yml

```diff
@@ -23,45 +23,52 @@ transport:
 verifier:
   name: inspec
 
+lifecycle:
+  post_verify:
+  - local: |
+      [[ ${KITCHEN_SAVE_IMAGE} == 'true' ]] || exit 0
+      docker_id=$(docker ps -a | grep ${KITCHEN_INSTANCE_NAME} | awk '{ print $1 }')
+      docker commit ${docker_id} pcluster-${KITCHEN_PHASE}/${KITCHEN_INSTANCE_NAME}:latest
```

Comment:
q: is KITCHEN_PHASE a known env variable?

Reply:
Yes, we set it in our helper script and we use it to understand which AMI to use (a bare one or a PCluster one).
This allows us to use a similar mechanism with any AMI we want to build on the fly.